npm-shrinkwrap.json

```diff
@@ -190,6 +190,12 @@
         "lodash": {
           "version": "3.10.1"
         },
+        "lodash.assign": {
```

Comment:
I think `lodash/assign` requires `lodash` to be installed, whereas `lodash.assign` is a standalone package. It should not change the size of the build in any way so I think you're right and I will update to `lodash` in the nest version!